StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/New Payment Method Screen/AddPaymentMethodViewController.swift

```diff
@@ -27,6 +27,12 @@ enum OverrideableBuyButtonBehavior {
 /// For internal SDK use only
 @objc(STP_Internal_AddPaymentMethodViewController)
 class AddPaymentMethodViewController: UIViewController {
+    enum AddPaymentMethodViewControllerError: Error {
```

Comment:
I'm wondering if "AddPaymentMethodViewControllerError" is unnecessary, since this is already nested. Like, the fully qualified type name that gets sent in analytics will be `AddPaymentMethodViewController. AddPaymentMethodViewControllerError`.  IMO it should either be just a nested `enum Error: Swift.Error` or put the error outside the class and keep the long name.